packages/hdwallet-metamask-shapeshift-multichain/src/shapeshift-multichain.ts

```diff
@@ -784,8 +780,13 @@ export class MetaMaskShapeShiftMultiChainHDWallet
   }
 
   public async ethSignMessage(msg: core.ETHSignMessage): Promise<core.ETHSignedMessage | null> {
-    const signedMessage = await this.ethGetAddress(this.provider);
-    return signedMessage ? Eth.ethSignMessage(msg, this.provider, signedMessage) : null;
+    const address = await this.ethGetAddress(this.provider);
+    return address ? Eth.ethSignMessage(msg, this.provider, address) : null;
+  }
```

Comment:
diff is drunk, this isn't diffed, but moved under, no idea why this was living in this random location above